### DISCUSSION TITLE
Unable to log in using oAuth after directus update to rc-95

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#8476] [Archive]
Console log:
POST https://t1553.ru/auth/refresh 400 (Bad Request)

Directus version 9.0.0.rc-95

**Comment by rijkvanzanten** [comment]
See https://github.com/directus/directus/issues/8378 / https://github.com/directus/directus/pull/8389

Bugs are better reported under Issue btw, this is more for questions/answers 🙂
